asset_dashboard/static/js/components/tables/AssetSearchTable.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import ReactTable from '../BaseTable'
+import PropTypes from 'prop-types'
+
+function AssetSearchTable({ rows }) {
+
+  const columns = [
+    {
+      Header: 'Identifier',
+      accessor: 'properties.identifier'
+    },
+    {
+      Header: 'Name',
+      accessor: 'properties.name', 
+    }
+  ]
```

Comment:
Can these live in `table_utils/assetSearchColumns.js` for consistency?

Reply:
Yep.